### PR TITLE
Docs 2850: Add Trustly direct to API reference

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/payment_methods/post-order-trustly.md [hunk 1]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+
+```json
+{
+  "type":"direct",
+  "order_id":"apitool_13557764",
+  "currency":"EUR",
+  "amount":1000,
+  "gateway":"TRUSTLY",
+  "description":"product description",
+  "custom_info":{
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:53]: Why is this empty object included in the request?

File: content/api/payment_methods/post-order-trustly.md [hunk 2]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
   }
 }
 ```
+> POST - /orders
+
+```json
+{
+  "type":"direct",
+  "order_id":"apitool_13557764",
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:48]:
```suggestion
  "order_id":"my-order-id-1",
```

File: content/api/payment_methods/post-order-trustly.md [hunk 3]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+      "phone2":"",
+      "state":null,
+      "zip_code":null
+    },
+    "description":"product description",
+    "fastcheckout":"NO",
+    "financial_status":"initialized",
+    "items":null,
+    "modified":"2021-11-08T13:14:05",
+    "order_id":"apitool_13557764",
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:108]:
```suggestion
    "order_id":"my-order-id-1",
```

File: content/api/payment_methods/post-order-trustly.md [hunk 4]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+  "order_id":"apitool_13557764",
+  "currency":"EUR",
+  "amount":1000,
+  "gateway":"TRUSTLY",
+  "description":"product description",
+  "custom_info":{
+    
+  },
+  "payment_options":{
+    "notification_url":"http://10.1.10.111/testtool/client/json-test/notification?type=notification",
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:57]:
```suggestion
    "notification_url":"https://www.example.com/client/notification?type=notification",
```

File: content/api/payment_methods/post-order-trustly.md [hunk 5]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+    "redirect_url":"http://10.1.10.111/testtool/client/json-test/notification?type=redirect",
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:58]:
```suggestion
    "redirect_url":"https://www.example.com/client/notification?type=redirect",
```

File: content/api/payment_methods/post-order-trustly.md [hunk 6]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+    "cancel_url":"http://10.1.10.111/testtool/client/json-test/notification?type=cancel"
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:59]:
```suggestion
    "cancel_url":"https://www.example.com/client/notification?type=cancel"
```

File: content/api/payment_methods/post-order-trustly.md [hunk 7]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+    "description":"product description",
+    "fastcheckout":"NO",
+    "financial_status":"initialized",
+    "items":null,
+    "modified":"2021-11-08T13:14:05",
+    "order_id":"apitool_13557764",
+    "payment_details":{
+      "account_holder_name":null,
+      "account_id":null,
+      "external_transaction_id":5095261,
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:112]:
```suggestion
      "external_transaction_id":1234567,
```

File: content/api/payment_methods/post-order-trustly.md [hunk 8]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+      "recurring_id":null,
+      "recurring_model":null,
+      "type":"TRUSTLY"
+    },
+    "payment_methods":[
+      {
+        "amount":1000,
+        "currency":"EUR",
+        "description":"product description",
+        "external_transaction_id":5095261,
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:123]:
```suggestion
        "external_transaction_id":1234567,
```

File: content/api/payment_methods/post-order-trustly.md [hunk 9]
```diff
@@ -40,13 +40,112 @@ meta_title: "API reference - Create a Trustly order - MultiSafepay Docs"
...
+        "payment_description":"Trustly",
+        "status":"initialized",
+        "type":"TRUSTLY"
+      }
+    ],
+    "reason":"",
+    "reason_code":"",
+    "related_transactions":null,
+    "status":"initialized",
+    "transaction_id":5095261,
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-trustly.md:133]:
```suggestion
    "transaction_id":1234567,
```

**Review by Kris-MultiSafepay — DISMISSED** [review]

File: content/api/payment_methods/post-order-trustly.md (hunk 1 above)
Comment (FrancescaNCook) [reply content/api/payment_methods/post-order-trustly.md]: Removed.

**Pull request merged**